Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -107,6 +108,17 @@ if (matchedFiles.length > 0) {
                 else { // copy
                     console.log(tl.loc('CopyingTo', file, targetPath));
                     tl.cp(file, targetPath);
+                    if (preserveTimestamp) {
+                        try {
+                            const fileStats = tl.stats(targetPath);
```

Comment:
Should this be `const fileStats: tl.FsStats = tl.stats(file);`? Right now we're getting the stats of targetPath and using that to set the stats at targetPath (which should do nothing).

I think its correct where you're doing it below.

Reply:
Thanks. You're right. Likely a copy/paste bug when moving things around. Fixed.